src/onegov/org/views/people.py

```diff
@@ -62,7 +63,9 @@ def visit_topics_with_people(
     }
 
 
-def person_functions_by_organization(subject_person, topics, request):
+def person_functions_by_organization(subject_person: Person,
+                                     topics: 'Iterator[Topic]',
```

Comment:
`Iterator` is a fairly strict protocol and is more commonly seen in return values rather than arguments (e.g. most generator functions and all generator comprehensions are `Iterator`s), `Iterable` is usually the better protocol, all it requires is a `__iter__` method, whereas `Iterator` also requires `__next__`.

Since you're not calling `next(topics)` anywhere `Iterable` is the correct thing to use.